HelpSource/Classes/PV_ChainUGen.schelp

```diff
@@ -1,52 +1,135 @@
 class:: PV_ChainUGen
-summary:: Base class for UGens that alter FFT chains
+summary:: Base class for UGens that operate on FFT chains
 categories:: UGens>FFT
 
 description::
-"PV_ChainUGen" is an abstract class - in other words, a class that you do not use directly. Instead, use one of its subclasses. It represents phase-vocoder UGens - i.e. UGens which apply some kind of transformation to the frequency-domain signal produced by FFT.
+code::PV_ChainUGen:: is an abstract class, not used directly, but only its subclasses. It represents phase-vocoder UGens - i.e. UGens which apply some kind of transformation to the frequency-domain signal produced by link::Classes/FFT::.
 
-(Strictly, this class encompasses all units whose output is an FFT chain. This is why FFT is in this group but IFFT is not - the IFFT unit outputs ordinary time-domain audio.)
+It encompasses all unit generators whose output is an FFT chain. This is why link::Classes/FFT:: is in this group but link::Classes/IFFT:: is not - the IFFT ugen outputs ordinary time-domain audio.
 
-For more info on using these units, see link::Guides/FFT-Overview::.
+For more information on using these unit generators, see link::Guides/FFT-Overview::.
 
 classmethods::
 private:: categories
 
 instancemethods::
+
+private::addCopiesIfNeeded
+
+method::fftSize
+Returns the FFT chain buffer's size.
+
+code::
+{ var chain = FFT(LocalBuf(1024)); chain.fftSize.poll; 0.0 }.play;
```

Comment:
i would prefer this split onto multiple lines

Reply:
ok, done.